cosmic-applet-audio/src/main.rs

```diff
@@ -25,6 +25,14 @@ use crate::pulse::DeviceInfo;
 use libpulse_binding::volume::VolumeLinear;
 
 pub fn main() -> cosmic::iced::Result {
+    let test_pulse = std::process::Command::new("pipewire-pulse")
+        .arg("--version")
+        .output();
+    if let Err(e) = test_pulse {
+        eprintln!("pipewire-pulse not found: {}", e);
+        return Ok(());
+    }
+
```

Comment:
Does it actually depend on pipewire-pulse and not just any pulseaudio server?
Can't we gracefully fail later, if our pulseaudio-crate doesn't manage to establish a connection?

A pulseaudio server could also be available over the network, so with appropriate environment variables set, no binary would need to exist on the system at all.